src/action/autopilot.js

```diff
@@ -3,20 +3,26 @@
  * whether autopilot should open channels.
  */
 
+import { checkHttpStatus } from '../helper';
+import * as log from './log';
+
 class AtplAction {
-  constructor(store, grpc, db, notification) {
+  constructor(store, grpc, info, db, notification) {
     this._store = store;
     this._grpc = grpc;
+    this._info = info;
     this._db = db;
     this._notification = notification;
   }
 
   /**
    * Initialize autopilot from the stored settings and enable it via grpc
-   * depending on if the user has enabled it in the last session.
+   * depending on if the user has enabled it in the last session. Fetch node
+   * scores are fetched from an api to inform channel selection.
    * @return {Promise<undefined>}
    */
   async init() {
+    await this.updateNodeScores();
```

Comment:
I believe we're supposed to be polling the bos scores every 1-2 hours to ensure they're up to date.

Reply:
Yeah. How often are the scores updated though in reality? The current solution updates them everytime the user starts the app, which is I think good enough for the beginning. We can add polling in the future if we really need it.